icons/tools/icons-build.js

```diff
@@ -1,17 +1,61 @@
-const generateFonts = require('./generateFonts');
-const generateCss = require('./generateCss');
-const generateSvgSprite = require('./generateSvgSprite');
-const processSvg = require('./processSvg');
-
-// generateCss();
-generateFonts()
-  .then(() => {
-    return generateCss();
-  })
-  .then(() => {
-    return generateSvgSprite();
-  })
-  .then(() => {
-    return processSvg();
-  });
+const { camelCase } = require("lodash");
+const handlebars = require("handlebars");
+const path = require("path");
+const { generateFonts } = require("@tbassetto/webfonts-generator");
 
+const {
+  compileCss,
+  generateFile,
+  generateIconsDataJson
+} = require("./generators");
+
+const FONT_NAME = "collab-ui-icons";
+const DEST = "fonts";
+const ICONS_SRC = "svg/*.svg";
+const DEFAULT_TEMPLATE_OPTIONS = {
+  baseSelector: ".icon",
+  classPrefix: "icon-",
+};
+
+(async function() {
+  try {
+    const result = await generateFonts(FONT_NAME, ICONS_SRC, DEST);
+
+    // Useful helpers for handlebars
+    handlebars.registerHelper("removeDashes", (selector, spacer) =>
+      selector.replace(/[_-]/g, spacer)
+    );
+    handlebars.registerHelper("equals", params => params[0] === params[1]);
+    handlebars.registerHelper("camelCase", str => camelCase(str));
+    handlebars.registerHelper("iosCharCode", str => "\\u{" + str + "}");
+
+    // Generate Scss files
+    const templateData = {
+      ...DEFAULT_TEMPLATE_OPTIONS,
+      ...result,
+      src: `url("#{$icon-font-path}/${path.parse(result.fontFiles.woff2).name}") format("woff2"),
+      url("#{$icon-font-path}/${path.parse(result.fontFiles.woff).name}") format("woff")`
```

Comment:
Note to myself: this code wrong, the file extensions are missing in the generated string. I'll fix it tomorrow.